docs/getting-started/experiments/cmabs/index.md

```diff
@@ -0,0 +1,84 @@
+# Contextual Bandits
+
+Contextual bandits (CMABs), similarly to [multi-armed bandits (MABs)](../mabs/index.md), are useful for running experiments where you have multiple variants of a feature / implementation that you want to test. However, the key difference is that contextual bandits take information about the end-user (e.g. gender, age, ethnicity) into account while converging to the best-performing variant.
+
+## What are Contextual Bandits (CMABs)?
+Contextual bandits work more or less in the same way as MABs do. You have $N$ variants of a feature / implementation, with corresponding probability of achieving a desired outcome. As you serve beneficiaries these variants and observe the outcome of their interaction with it, you update these probabilities.
+
+The strategy for updating these probabilities is also similar: you can optimize for exploration or exploitation (*though our current implementation only supports Thompson sampling -- we're working on adding more!*). Crucially, we take user information into account while updating these probabilities for contextual bandits. Thus, rather than having a single best-performing variant at the end of an experiment, you instead have the best-performing variant that depends on the user context.
```

Comment:
```suggestion
The strategy for updating these probabilities is also similar. The crucial difference is that we take user information into account while updating these probabilities for contextual bandits. Thus, rather than having a single best-performing variant at the end of an experiment, you instead have the best-performing variant that depends on the user context.
```